Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ObservableGroup/ObservableGroupPage.xaml

```diff
@@ -0,0 +1,75 @@
+﻿<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.ObservableGroupPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:collections="using:Microsoft.Toolkit.Collections"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:system="using:System"
+      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <Style TargetType="Button">
+            <Setter Property="Margin" Value="8,0,0,0" />
+        </Style>
+
+        <DataTemplate x:Key="PersonDataTemplate"
+                      x:DataType="local:Person">
+            <TextBlock Text="{x:Bind Name}" />
+        </DataTemplate>
+
+        <DataTemplate x:Key="GroupDataTemplate"
+                      x:DataType="collections:IReadOnlyObservableGroup">
+            <StackPanel Orientation="Horizontal">
+                <TextBlock Style="{StaticResource SubtitleTextBlockStyle}"
+                           Text="{x:Bind (x:String)Key}" />
+                <TextBlock Margin="8,0,0,0"
+                           VerticalAlignment="Bottom"
+                           Opacity="0.8"
+                           Style="{StaticResource CaptionTextBlockStyle}"
+                           Text="{x:Bind system:String.Format('{0} item(s)', Count), Mode=OneWay}" />
+            </StackPanel>
+        </DataTemplate>
+
+        <CollectionViewSource x:Key="cvs"
+                              x:Name="cvs"
+                              IsSourceGrouped="True"
+                              Source="{x:Bind Contacts}" />
+    </Page.Resources>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <StackPanel Margin="0,4"
+                    HorizontalAlignment="Center"
+                    Orientation="Horizontal">
+            <TextBox x:Name="NewContact"
```

Comment:
Hitting `Enter` on this control should activate the Add button and clear the text, it'd be easy to quickly create a few items this way via keyboard only.